app/_src/guides/progressively-rolling-in-strict-mtls.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Progressively rolling in strict mTLS
+---
+
+The [MeshTLS](/docs/{{ page.version }}/policies/meshtls/) policy allows you to gradually migrate services to mutual TLS without dropping a packet.
+
+## Prerequisites
+- Completed [quickstart](/docs/{{ page.version }}/quickstart/kubernetes-demo/) to set up a zone control plane with demo application
+
+## Gradually bring another service into the mesh
+
+### Start a second demo app
+
+First we start a new demo setup in another namespace called `kuma-demo-migration`.
+The below command installs the demo one more time in another namespace:
+
+```bash
+curl -s https://raw.githubusercontent.com/kumahq/kuma-counter-demo/master/demo.yaml | 
+  sed "s#kuma.io/sidecar-injection: enabled#kuma.io/sidecar-injection: disabled#" |
+  sed "s#name: kuma-demo#name: kuma-demo-migration#" |
+  sed "s#redis.kuma-demo.svc.cluster.local#redis.kuma-demo-migration.svc.cluster.local#" |
+  sed "s#namespace: kuma-demo#namespace: kuma-demo-migration#" | kubectl apply -f -
+```
+
+Below diagram shows which applications are inside the mesh and which are not:
+
+{% mermaid %}
+---
+title: service graph of the second demo app
+---
+flowchart LR
+    subgraph meshed
+        subgraph kuma-demo
+            direction LR
+            demo-app(demo-app :5000)
+            redis(redis :6379)
+            demo-app --> redis
+        end
+    end
+    subgraph non-meshed
+        subgraph kuma-demo-migration
+            direction LR
+            redis2(redis :6379)
+            demo-app2(demo-app :5000)
+            demo-app2 --> redis2
+        end
+    end
+{% endmermaid %}
+
+### Enable port forwarding for both second app
+
+```bash
+kubectl port-forward svc/demo-app -n kuma-demo-migration 5001:5000
+```
+
+Open up both apps' GUI and turn on auto incrementing.
+
+### Enable permissive mode on redis
+
+We begin with preparing redis to start in [permissive](/docs/{{ page.version }}/policies/meshtls/#configuration) mode when deployed inside the mesh.
+To enable permissive mode we define this `MeshTLS` policy:
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: MeshTLS
+metadata:
+  name: redis
+  namespace: kuma-system
+  labels:
+    kuma.io/mesh: default
+spec:
+  targetRef:
+    kind: MeshSubset
+    tags:
+      app: redis
+      k8s.kuma.io/namespace: kuma-demo-migration
+  from:
+  - targetRef:
+      kind: Mesh
+    default:
+      mode: Permissive
```

Comment:
I think in other guides we take stuff that can be directly copy pasted. Also I think the namespace should be `{{ site.namespace }}` e.g: https://github.com/kumahq/kuma-website/blob/master/app/_src/guides/gateway-builtin.md#define-a-route-using-meshhttproute

Reply:
yes, definitely!